src/database/search_handler.cc

```diff
@@ -115,7 +115,7 @@ std::unique_ptr<SearchToken> SearchLexer::nextToken()
     return nullptr;
 }
 
-std::string SearchLexer::getQuotedValue(const std::string& input)
+std::string SearchLexer::getQuotedValue(std::string_view input)
```

Comment:
It's a string already, so `string_view` creates a new object instead of passing the existing object.